ci/workflows/pull-request.yml

```diff
@@ -762,3 +753,20 @@ steps:
           - BUILDKITE_TOKEN
           - GITHUB_TOKEN
     timeout_in_minutes: 5
+
+  - wait: ~
+
+  - label: "upload coverage reports"
+    command: "ci/scripts/upload-coverage.sh"
+    # TODO: shall we skip this if `run-selected` or `CI_STEPS` is specified.
```

Comment:
I think an immediate goal is to let us evaluate new features. So keeping `run-selected` makes sense, since new features may add test only to `main-cron`, rather than to pr, if the test is heavy.